composer.json

```diff
@@ -42,6 +42,7 @@
         ],
         "cs-check": "phpcs --colors -p --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests",
         "cs-fix": "phpcbf --colors --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests",
-        "test": "phpunit --colors=always"
+        "test": "phpunit --colors=always",
+        "coverage": "App\\Console\\Coverage::generateCoverage"
```

Comment:
should we name it

    "test-coverage": ...

to be more precise that this is coverage for tests?